src/schemas.py

```diff
@@ -45,7 +45,7 @@ class PeriodOfTime(BaseModel):
 class Column(BaseModel):
     name: str
     datatype: str
-    titles: str
+    title: str
```

Comment:
needs to be title**s** plural, this is 100% my fault as I seem to have changed it in versions.json when I shouldnt have, my bad. As it stands this change makes the version tests pass but makes the edition tests fail (as they both use `Column` and are currently different.

just change it to title**s** plural in the versions.json and this back as well and we're good - make sure `make test` passes though, just in case.